package.json

```diff
@@ -106,17 +122,26 @@
     "styled-components": "^6.0.4",
     "supabase": "^1.142.2",
     "typescript": "^5.5.3",
+    "util": "^0.12.5",
```

Comment:
Is this required? I think it's part of node by default

Reply:
Removed it and it still works fine, so not required